### PR TITLE
Register ContMechTensors [https://github.com/KristofferC/ContMechTensors.jl.git]: v0.1.0

### PR DESCRIPTION
**Pull request by KristofferC**
Trying again :)

### CODE REVIEW

File: ContMechTensors/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline ContMechTensors/versions/0.0.1/requires]: what minimum version?

Comment (KristofferC) [inline ContMechTensors/versions/0.0.1/requires]: Doesn't matter. I will not go back and retag and update minimum version for an obsolete Julia version.

Comment (tkelman) [inline ContMechTensors/versions/0.0.1/requires]: And if you do want to tag a version that supports 0.4, it's not necessary to re-tag or change the sha to modify version requirements, this file can be changed in place just in metadata.

Comment (KristofferC) [inline ContMechTensors/versions/0.0.1/requires]: I deleted the old tag and removed the version from the METADATA pr.

**Pull request merged**